src/state.rs

```diff
@@ -539,7 +540,7 @@ pub fn log_header<W: Write>(writer: &mut W, seed: u32) {
     writeln!(writer, "{}", crate::metadata::GIT_HASH).unwrap();
 }
 
-pub fn log_command<W: Write>(writer: &mut W, command: Command) {
+pub fn log_command<W: Write>(writer: &mut W, command: &Command) {
```

Comment:
I want to keep this as is. Most of the commands used in the game are tiny and I don't mind cloning the occasional string.

This just adds visual noise.